pkg/operator/k8sutil/deployment.go

```diff
@@ -24,10 +24,13 @@ import (
 	"github.com/rook/rook/pkg/util"
 	apps "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 )
 
+const osdOverPVCLabelKey = "ceph.rook.io/pvc"
```

Comment:
This is in the `k8sutil` package, which is for all storage providers. We can't add an osd label here. How about we pass it in to the helper method?

Reply:
cc @ashishranjan738

Reply:
changed